site/tests/RepoCrawler/WebNotificationConsumerTest.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+namespace App\Tests\RepoCrawler;
+
+use App\Consumer\WebNotificationConsumer;
+use App\Entity\User;
+use App\Util\Notification;
+use Mgilet\NotificationBundle\Manager\NotificationManager;
+use PhpAmqpLib\Message\AMQPMessage;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Tests for the Web Notification Consumer
+ *
+ * @author Aquib Baig <aquibbaig97@gmail.com>
+ */
+class WebNotificationConsumerTest extends TestCase
+{
+    protected $notification;
+
+    /**
+     * Called before every test
+     */
+    public function setUp()
+    {
+        $this->notification = new Notification();
+        $this->notification->setMessage('Your project was added to LibreCores')
+            ->setSubject('New Project')
+            ->setType('new_project');
+        $user = new User();
+        $user->setUsername('test');
+        $this->notification->setRecipient($user);
+
+        // Assert if fields are set properly
+        $this->assertEquals('Your project was added to LibreCores', $this->notification->getMessage());
+        $this->assertEquals('New Project', $this->notification->getSubject());
+        $this->assertEquals('new_project', $this->notification->getType());
+        $this->assertEquals('test', $this->notification->getRecipient()->getUsername());
+    }
+
+    /**
+     * Called after every test
+     */
+    public function tearDown()
+    {
+        $this->notification = [];
```

Comment:
Why do you set the notification variable to an array?

Reply:
I have set it to null now